tensorflow/contrib/py2tf/converters/side_effect_guards.py

```diff
@@ -34,6 +34,8 @@
 from __future__ import division
 from __future__ import print_function
 
+import contextmanager
```

Comment:
Also remove changes from this file.
As py2tf does a lot of templating and on the fly evaluation of the code, the linter errors here are misleading.